src/assets/drizzle/styles/utility/_toolkit-overrides.scss

```diff
@@ -47,6 +47,10 @@
   margin-top: -2px;
 }
 
-.drizzle-b-Body.sprk-u-Overflow--hidden {
+.#{$app-namespace}-b-Body.sprk-u-Overflow--hidden {
   overflow: auto !important; // this is to override a utility class (sprk-u-Overflow--hidden) that is added by JS.
 }
+
+.#{$app-namespace}-c-Preview .sprk-c-Modal {
```

Comment:
Fixes bug where modals were behind main spark site nav.

Reply:
https://github.com/sparkdesignsystem/spark-design-system/issues/474

Reply:
also ensures they are above the spark site copy code buttons (bug: https://github.com/sparkdesignsystem/spark-design-system/issues/473)